apollo-backend/src/test/java/io/logz/apollo/BlockerTest.java

```diff
@@ -48,6 +48,29 @@ public void testEverythingBlocker() throws Exception {
         ModelsGenerator.createAndSubmitDeployment(apolloTestClient, environment, service, deployableVersion);
     }
 
+    @Test
+    public void testUnconditionalBlockerWithException() throws Exception {
+        ApolloTestClient apolloTestClient = Common.signupAndLogin();
+        ApolloTestAdminClient apolloTestAdminClient = Common.getAndLoginApolloTestAdminClient();
+
+        Environment environment = ModelsGenerator.createAndSubmitEnvironment(apolloTestClient);
+        Service service = ModelsGenerator.createAndSubmitService(apolloTestClient);
+        DeployableVersion deployableVersion = ModelsGenerator.createAndSubmitDeployableVersion(apolloTestClient, service);
+
+        List<Integer> exceptionServiceIds = new ArrayList<Integer>() {{ add(service.getId()); }};
+        List<Integer> exceptionEnvironmentIds = new ArrayList<Integer>() {{ add(environment.getId()); }};
```

Comment:
Please split the test into 3 - 
Block all but service
Block all but environment
Block all but service in a specific environment